src/ui_ng/src/app/project/project.component.ts

```diff
@@ -63,6 +63,15 @@ export class ProjectComponent implements OnInit, OnDestroy {
   subscription: Subscription;
   loading: boolean = true;
 
+  get selecteType (): number {
```

Comment:
_selectProjectType: number;

get selectType(): number {
  return this._selectProjectType;
}

set selectType(v: number) {
  this._selectProjectType = v;
  if (window.sessionStorage) {
       window.sessionStorage['projectTypeValue'] = v;
     }
}


....


doFilterProjects(){
  this.selectType
}